packages/@aws-cdk/aws-dynamodb-global/lib/lambda/handler.js

```diff
@@ -0,0 +1,105 @@
+exports.handler = function(event, context, callback) {
+    var AWS = require('aws-sdk');
+    // Install watchdog timer as the first thing
+    setupWatchdogTimer(event, context, callback);
+    console.log("REQUEST RECEIVED:\n" + JSON.stringify(event));
+    if (event.RequestType === "Create") {
+      console.log("CREATE!");
+      var dynamodb = new AWS.DynamoDB();
+      var regions = [];
+      for (var i = 0; i < event.ResourceProperties.Regions.length; i++) {
+        regions.push({
+          "RegionName": event.ResourceProperties.Regions[i]
+        });
+      }
+      var params = {
+        GlobalTableName: event["ResourceProperties"]["TableName"],
+        ReplicationGroup: regions
+      };
+      console.log(params);
+      dynamodb.createGlobalTable(params, function(err, data) {
+        if (err) {
+          console.log(err, err.stack); // an error occurred
+          sendResponse(event, context, "FAILED");
+        } 
+        else {
+          console.log(data);           // successful response
+          sendResponse(event, context, "SUCCESS", { Message: "Resource creation successful!" });
+        }
+      });
+    } else if (event.RequestType === "Update") {
+      console.log("UDPATE!");
+      // Put your custom update logic here
+      sendResponse(event, context, "SUCCESS", { Message: "Resource update successful!" });
+    } else if (event.RequestType === "Delete") {
+      console.log("DELETE!");
+      // Put your custom delete logic here
+      sendResponse(event, context, "SUCCESS", { Message: "Resource deletion successful!" });
+    } else {
+      console.log("FAILED!");
+      sendResponse(event, context, "FAILED");
+    }
```

Comment:
You must catch all exceptions and send a FAILED response. Otherwise, any exception will cause the CFN deployment to timeout after 4 hours.

Then, all failure cases should just "throw".

Ideally, if you can also have only a single point in your code that sends a SUCCESS response, it would be safer, especially if someone updates this code in the future and forgets to handle some case (alternatively, "throw" when you fall through).